pkg/alerter/alerter.go

```diff
@@ -0,0 +1,79 @@
+package alerter
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/sirupsen/logrus"
+)
+
+type Alerter struct {
+	sync.Mutex
+	Alerts   Alerts
+	AlertsCh chan Alerts
+}
+
+func NewAlerter() *Alerter {
+	return &Alerter{
+		Alerts:   make(Alerts),
+		AlertsCh: make(chan Alerts),
+	}
+}
+
+func (a *Alerter) Start() {
+	go func() {
+		defer close(a.AlertsCh)
+		for {
+			alert := <-a.AlertsCh
+			a.mergeAlerts(alert)
+		}
+	}()
+}
+
+func (a *Alerter) Alert(key string, alert Alert) {
```

Comment:
In term of naming, I found `Add` or `NewAlert` / `AddAlert` more clear (the second option is maybe a little redundant `alerter.AddAlert`)

Reply:
I find `Add` very minimalist and don't incorporate the idea of sending alert. What about `SendAlert` or `Send` ?

Reply:
I like SendAlert too

Reply:
👍🏻 for SendAlert

Reply:
Done 7f5211c